doc/HorizontalReshardingGuide.md

```diff
@@ -0,0 +1,127 @@
+# Horizontal resharding guide
+
+This is a step-by-step "cookbook" style guide of instructions to follow in order to split a single-sharded keyspace into two.
+
+## Preparing the source shard
+
+Let’s assume that you’ve already got a keyspace up and running, with a single shard (the default case). For example, you might have the following keyspace/shard: `test_keyspace/0`. It’s also a requirement that you have at least one (but preferably two) rdonly tablets for the source shard.
```

Comment:
Yep, that's true, this is the only step that is different if you already have multiple shards. Added a note to that effect.